src/server/pages/signin.js

```diff
@@ -14,23 +14,12 @@ export default function signin ({ csrfToken, providers, callbackUrl, email, erro
     return false
   })
 
-  const errors = {
-    Signin: 'Try signing with a different account.',
-    OAuthSignin: 'Try signing with a different account.',
-    OAuthCallback: 'Try signing with a different account.',
-    OAuthCreateAccount: 'Try signing with a different account.',
-    EmailCreateAccount: 'Try signing with a different account.',
-    Callback: 'Try signing with a different account.',
-    OAuthAccountNotLinked: 'To confirm your identity, sign in with the same account you used originally.',
-    EmailSignin: 'Check your email address.',
-    CredentialsSignin: 'Sign in failed. Check the details you provided are correct.',
-    default: 'Unable to sign in.'
-  }
-
-  const error = errorType && (errors[errorType] ?? errors.default)
+  const error = errorType && (texts.errors[errorType] ?? texts.errors.default)
 
   return (
     <div className='signin'>
+      {locale &&
+        <p>Locale: {locale || 'N/A'}</p>}
```

Comment:
I don't think we would actually need this. Especially with a hardcoded `Locale:` English text. What was the intention here?

Reply:
Added these just for testing purposes, will remove them.